python/cucim/tests/util/gen_tiff.py

```diff
@@ -58,7 +58,11 @@ def save_image(
 
         compression = COMPRESSION_MAP.get(compression)
```

Comment:
Maybe this can just be a default before

```suggestion
        compression = COMPRESSION_MAP.get(compression)
        compressionargs = None
```